pkg/collector/corechecks/net/ntp.go

```diff
@@ -34,6 +42,18 @@ var (
 
 	tlmNtpOffset = telemetry.NewGauge("check", "ntp_offset",
 		nil, "Ntp offset")
+
+	awsNTPHosts     = []string{"169.254.169.123"}
+	gcpNTPHosts     = []string{"metadata.google.internal"}
+	azureNTPHosts   = []string{"time.windows.com"}
+	alibabaNTPHosts = []string{
+		"ntp.cloud.aliyuncs.com", "ntp1.cloud.aliyuncs.com", "ntp2.cloud.aliyuncs.com", "ntp3.cloud.aliyuncs.com",
+		"ntp4.cloud.aliyuncs.com", "ntp5.cloud.aliyuncs.com", "ntp6.cloud.aliyuncs.com", "ntp7.cloud.aliyuncs.com",
+		"ntp8.cloud.aliyuncs.com", "ntp9.cloud.aliyuncs.com", "ntp10.cloud.aliyuncs.com", "ntp11.cloud.aliyuncs.com",
+		"ntp12.cloud.aliyuncs.com",
+	}
+	tencentNTPHosts = []string{"ntpupdate.tencentyun.com"}
+	datadogNTPHosts = []string{"0.datadog.pool.ntp.org", "1.datadog.pool.ntp.org", "2.datadog.pool.ntp.org", "3.datadog.pool.ntp.org"}
```

Comment:
This info should be contained in each cloud provider package. Each of this package should encapsulate all the knowledge we have about each cloud provider.